src/apps/material/material.dev.tsx

```diff
@@ -824,6 +829,11 @@ export default {
       name: "Open order unknown user text",
       defaultValue: "User not found",
       control: { type: "text" }
+    },
+    openOrderErrorMissingPincodeText: {
```

Comment:
This needs to be added to dpl-cms as well, right?

Reply:
https://reload.zulipchat.com/#narrow/stream/240325-DDF/topic/Open.20Order.20tekst.20strenge

Reply:
I don't get it, it's not there.